Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>Maybe we can output all of the field sites that were in the input? That would make it easier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>